moto/ec2/models/availability_zones_and_regions.py

```diff
@@ -64,33 +66,101 @@ class RegionsAndZonesBackend:
 
     zones = {
         "af-south-1": [
-            Zone(region_name="af-south-1", name="af-south-1a", zone_id="afs1-az1"),
-            Zone(region_name="af-south-1", name="af-south-1b", zone_id="afs1-az2"),
-            Zone(region_name="af-south-1", name="af-south-1c", zone_id="afs1-az3"),
```

Comment:
These changes are just cosmetic, right? I realize it would be a pain to undo, but keeping them as-is would make this a lot easier to review..

Reply:
Yes. I've reverted it to as it was.